lumigator/python/mzai/backend/backend/api/routes/jobs.py

```diff
@@ -25,7 +26,7 @@ def create_inference_job(
 @router.post("/evaluate/", status_code=status.HTTP_201_CREATED)
 def create_evaluation_job(
     service: JobServiceDep,
-    request: JobCreate,
+    request: JobEvalCreate,
```

Comment:
@aittalam fyi heads up here's the scope of work I've been doing that parallels yours. Not ready yet, just an idea of where I'm headed